source/command/atlas-kubernetes.txt

```diff
@@ -40,6 +40,7 @@ Related Commands
 ----------------
 
 * :ref:`atlas-kubernetes-config` - Manage Kubernetes configuration resources.
+* :ref:`atlas-kubernetes-dry-run` - Deploy and run Atlas Kubernetes Operator in dry-run mode
```

Comment:
in dry run mode

Also, can't comment but it seems that these bullets could be written down without the period at the end -- all of them, on lines 42-44.